docs/src/references/mappers/mqtt-topics.md

```diff
@@ -52,7 +52,7 @@ The topics follow the below format
     c8y/error
 
 You can find more information about Cumulocity topics
-[Here](https://tech.forums.softwareag.com/t/cumulocity-iot-tips-and-tricks-mqtt-cheat-sheet/237187)
+[Here](https://cumulocity.com/docs/smartrest/mqtt-static-templates/)
```

Comment:
We also have to update other document links referring to old Cumulocity guide at some point..

Reply:
There shouldn't be any other links now in the repo (or not that I could see)

Reply:
My word was confusing sorry. I mean all links https://cumulocity.com/guides/ should be updated to https://cumulocity.com/docs/. I see several usages on our docs. However, this can be addressed in a separated PR for sure as the purpose is a bit different.

Reply:
I'll create a follow-up PR